src/vs/workbench/contrib/debug/common/debug.ts

```diff
@@ -337,7 +337,7 @@ export interface IBaseBreakpoint extends IEnablement {
 	readonly hitCondition?: string;
 	readonly logMessage?: string;
 	readonly verified: boolean;
-	readonly idFromAdapter: number | undefined;
+	idFromAdapter(sessionId: string): number | undefined;
```

Comment:
I would prefer if this method is called `getIdFromAdapter` that way it would be more clear that it is a method, and not an attribute on the class.

Reply:
Done.